test/zzz-rsyslog.sh

```diff
@@ -178,8 +178,8 @@ fi
 $deployfunc
 
 # MERGE_JASON_LOG has to be true?
-oc set env $rsyslog_ds MERGE_JSON_LOG=true
-rpod=$( get_running_pod rsyslog )
+#oc set env $rsyslog_ds MERGE_JSON_LOG=true
+#rpod=$( get_running_pod rsyslog )
```

Comment:
yeah - I don't see any processing that requires `MERGE_JSON_LOG=true`

Reply:
This [1] (and equivalent $!log) is the only places MERGE_JSON_LOG has an impact.  And my CLO PR disabled MERGE_JSON_LOG, which looked the biggest change other than mmexternal.  

I verified even if mmexternal is disabled by setting USE_MMEXTERNAL=false, zzz-rsyslog.sh failed so I was trying MERGE_JSON_LOG next.  But before running the test with MERGE_JSON_LOG=true, I found the "unbound variable" error, so I'm testing the fix now.  Sorry, very slow move...

[1] - snippet of 65-viaq-formatting.conf
```
 11     set $!pipeline_metadata!collector!original_raw_message = $!MESSAGE;
 12     if (strlen(`echo $MERGE_JSON_LOG`) > 0) and (`echo $MERGE_JSON_LOG` == "true") then {
 13         action(type="mmnormalize" ruleBase="/etc/rsyslog.d/parse_json.rulebase" variable="$!MESSAGE")
 14     }
```